### PR TITLE
implemented CinemaHallController

### PR DESCRIPTION
**Pull request by matviichuk92**

### CODE REVIEW

**Review by pavlokhomiak — APPROVED** [review]
good!

File: src/main/java/com/cinema/controller/OrderController.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        this.orderDtoMapper = orderDtoMapper;
+        this.orderService = orderService;
+        this.userService = userService;
+        this.shoppingCartService = shoppingCartService;
+    }
+
+    @PostMapping("/complete")
+    public void completeOrder(@RequestParam Long userId) {
+        orderService.completeOrder(userService.get(userId),
+                shoppingCartService.getByUser(userService.get(userId)).getTickets());
```
Comment (KseniiaMakarova) [inline src/main/java/com/cinema/controller/OrderController.java]: local variables would be nice here
this is hard to read

File: src/main/java/com/cinema/model/dto/TicketResponseDto.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package com.cinema.model.dto;
+
+public class TicketResponseDto {
+    private Long id;
+    private MovieSessionResponseDto session;
```
Comment (KseniiaMakarova) [inline src/main/java/com/cinema/model/dto/TicketResponseDto.java]: the `tickets` list in Cart/OrderResponseDto becomes more and more complex

let's have some really simple fields in this TicketResponseDto
or return `List<Long> ticketIds` in Cart/OrderResponseDto

File: src/main/java/com/cinema/model/dto/mapper/TicketDtoMapper.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+@Component
+public class TicketDtoMapper {
+    private final MovieSessionDtoMapper movieSessionDtoMapper;
+
+    public TicketDtoMapper(MovieSessionDtoMapper movieSessionDtoMapper) {
+        this.movieSessionDtoMapper = movieSessionDtoMapper;
+    }
+
+    public TicketResponseDto mapTicketToDto(Ticket ticket) {
```
Comment (KseniiaMakarova) [inline src/main/java/com/cinema/model/dto/mapper/TicketDtoMapper.java]:
```suggestion
    public TicketResponseDto mapToDto(Ticket ticket) {
```
uniform method names

File: src/main/java/com/cinema/model/dto/mapper/TicketDtoMapper.java [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+        TicketResponseDto ticketResponseDto = new TicketResponseDto();
+        ticketResponseDto.setId(ticket.getId());
+        ticketResponseDto.setSession(movieSessionDtoMapper.mapToSession(ticket.getMovieSession()));
```
Comment (KseniiaMakarova) [inline src/main/java/com/cinema/model/dto/mapper/TicketDtoMapper.java]:
```suggestion
        ticketResponseDto.setSession(movieSessionDtoMapper.mapToDto(ticket.getMovieSession()));
```
it is just for you to notice - above I suggested to remove this from here

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

File: src/main/java/com/cinema/model/dto/OrderResponseDto.java [hunk 5]
```diff
@@ -5,7 +5,7 @@
 
 public class OrderResponseDto {
     private Long orderId;
-    private List<TicketResponseDto> tickets;
+    private List<Long> ticketsId;
```
Comment (KseniiaMakarova) [inline src/main/java/com/cinema/model/dto/OrderResponseDto.java]: `ticketIds` in both classes is more informative

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request merged**